src/pages/support.tsx

```diff
@@ -138,17 +143,17 @@ const Support: React.FC<{ patrons: string[]; patreonFunding: number }> = ({
           buttonText="contact us"
           buttonHref="mailto:hello@veganhacktivists.org"
         >
-          For tax-deductible (U.S.), larger donations
+          For larger donations (US tax-deductible)
         </DonationCard>
       </div>
       <Paragraph>
         We prefer a{' '}
         <span className="font-bold">monthly donation via Patreon</span> as that
-        gives us the most stability every month, but we also accept one time
-        donations via Paypal. If you would like to make a larger contribution of
-        $1000 or more (thank you!) please contact us for other options like wire
-        or check. We have a fiscal sponsor that will let you claim your donation
-        as tax deductable!
+        gives us the most stability every month, but we also accept one-time
+        donations via PayPal. If you would like to make a larger contribution of
+        $1,000 or more (thank you!), please{' '}
+        <Link href="mailto:hello@veganhacktivists.org">contact us</Link> for
```

Comment:
Is there a way to make this link stand out? 

It currently blends in with the rest of the text so it's not clear that people can click on it. Nbd if not, just a nice-to-have.

Reply:
Sure! We have a `<CustomLink>` component that does that (with a predefined style), and if you want something different you can add an `<a>` element:

```
<Link>
    <a add styles here!>
        Text
    </a>
</Link>
```

Reply:
Thank you, updated!